src/main/java/seedu/address/model/task/exceptions/TaskNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package seedu.address.model.task.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified task.
+ */
+public class TaskNotFoundException extends RuntimeException {
+}
```

Comment:
Is this method supposed to be empty?

Reply:
Yes. Is just a more detailed type of exception, so it doesn't need any implementation